src/main/webapp/app/exercises/file-upload/assess/file-upload-assessment-dashboard.component.ts

```diff
@@ -150,4 +152,19 @@ export class FileUploadAssessmentDashboardComponent implements OnInit {
     public durationString(completionDate: Date, initializationDate: Date) {
         return this.momentDiff.transform(completionDate, initializationDate, 'minutes');
     }
+
+    public sortRows() {
+        if (this.predicate === 'durationInMinutes') {
+            this.submissions.forEach((submission) => (submission.durationInMinutes = this.durationForSubmission(submission)));
```

Comment:
Does it make sense to compute this each time someone clicks on sort? I would rather suggest to compute this after the data was loaded from the server

Reply:
it is now done in the component when the submissions are loaded

Reply:
Addressed via: 55d5b750d42c0b30ad72ac670986a8ec81136426